src/Promitor.Core/Metrics/MetricAggregationTypeConverter.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using Azure.Monitor.Query.Models;
+using Microsoft.Azure.Management.Monitor.Fluent.Models;
+
+namespace Promitor.Core.Metrics;
+
+public static class MetricAggregationTypeConverter 
```

Comment:
Just to confirm, did we add unit tests for these?

Reply:
Not previously but I added them just now :)